src/components/ModalHelp.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { isTouch } from '../browser'
+import { store } from '../store'
```

Comment:
I'll accept this, but note that importing the `store` directly into a component is considered a bad practice. It is recommended to access the store through `mapStateToProps` or the `useSelector` hook.

Reply:
Sure. Thanks for reviewing my code!